technical-resources/protocol-overview/smart-contract-addresses.md

```diff
@@ -154,7 +163,6 @@
 | WithdrawalManager (Proxy) | [`0x7f0d63e2250bc99f48985b183af0c9a66bbc8ac3`](https://etherscan.io/address/0x7f0d63e2250bc99f48985b183af0c9a66bbc8ac3) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
 | PoolDelegateCover (Proxy) | [`0x4Ab7F6F8DE3FE2575806a21F9Cac630DD51a84Af`](https://etherscan.io/address/0x4Ab7F6F8DE3FE2575806a21F9Cac630DD51a84Af) | [`proxy-factory @ v1.0.0`](https://github.com/maple-labs/proxy-factory/releases/tag/v1.0.0) |
 
-
 ### Maven11 WETH Pool Contracts
```

Comment:
You are still missing the new opportunistic pool, please double check you have added all the correct pools

Reply:
Added the opportunistic pool, noticed we're missing the old `laser` and `cicada` pools, are they relevant anymore?
Most of the pools on this list are deprecated or inactive.

Reply:
Lets add them so the list is exhaustive, worth having incase the frontend stops supporting a deprecated pool and someone needs the address to hit etherscan directly.